src/rime/config/config_compiler.h

```diff
@@ -43,6 +43,8 @@ class ConfigCompiler {
  public:
   static constexpr const char* INCLUDE_DIRECTIVE = "__include";
   static constexpr const char* PATCH_DIRECTIVE = "__patch";
+  static constexpr const char* APPEND_DIRECTIVE = "__append";
+  static constexpr const char* MERGE_DIRECTIVE = "__merge";
```

Comment:
`__merge` never appears in the tests / sample YAML?

Reply:
`__merge` can be used for maps the same way as `__append` is for lists. 
usually using `/+` or implicit merging will be enough. I cannot find a case where `__merge` must be used.